simSPI/linear_simulator/shift_utils.py

```diff
@@ -0,0 +1,90 @@
+"""Module containing method to shift an image in Fourier domain."""
```

Comment:
NIT: Module containing a method to apply a spatial shift on an image in Fourier domain.